ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_bfd.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from functools import cached_property
+
+from .utils import UtilsMixin
+
+
+class RouterBfdMixin(UtilsMixin):
+    """
+    Mixin Class used to generate structured config for one key.
+    Class should only be used as Mixin to a AvdStructuredConfig class
+    """
+
+    @cached_property
+    def router_bfd(self) -> dict | None:
+        """
+        return structured config for router_bfd
+        """
+        if self._bfd_multihop is None:
+            return None
+
+        return {
```

Comment:
all this could just be a copy of the input variable. There is nothing added or removed here. You should also shave off `None` values

Reply:
Addressed in 6110996